client/src/main/java/io/getstream/chat/android/client/helpers/AttachmentHelper.kt

```diff
@@ -6,8 +6,8 @@ import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
 
 class AttachmentHelper(private val systemTimeProvider: SystemTimeProvider = SystemTimeProvider()) {
 
-    fun hasValidUrl(attachment: Attachment): Boolean {
-        val url = attachment.url?.toHttpUrlOrNull() ?: return false
+    fun hasValidImageUrl(attachment: Attachment): Boolean {
+        val url = attachment.imageUrl?.toHttpUrlOrNull() ?: return false
```

Comment:
What if attachment is not an image? Shouldn't we also validate `Attachment:: assetUrl `?

Reply:
It could be done, but I don't see any reason. We want to use old image urls only for performance issues